client/src/components/forms/DepositBoxForm/DepositBoxForm.tsx

```diff
@@ -20,37 +21,25 @@ import { CalculatorView } from '@/components/Calculator/View/CalculatorView';
 import { Dispatch, SetStateAction, useEffect, useState } from 'react';
 import { Spinner } from '@components/Layout/Spinner/Spinner';
 import { FormMessage, GIVE_BOX_WRONG_ID_ERROR_RESPONSE, NetworkError } from '@/utils';
+import { moneyValuesType, useSum } from './useSum';
+import { uid } from 'uid';
 
-const moneyValues = {
-  '1gr': 0.01,
-  '2gr': 0.02,
-  '5gr': 0.05,
-  '10gr': 0.1,
-  '20gr': 0.2,
-  '50gr': 0.5,
-  '1zl': 1,
-  '2zl': 2,
-  '5zl': 5,
-  '10zl': 10,
-  '20zl': 20,
-  '50zl': 50,
-  '100zl': 100,
-  '200zl': 200,
-  '500zl': 500,
-  EUR: 4.71,
-  GBP: 5.37,
-  USD: 4.33,
+const moneySlice = {
```

Comment:
It'd be a good idea to leave a comment explaining what these moneySlice values means